plinic/src/components/input/Input.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
```

Comment:
![image](https://user-images.githubusercontent.com/87893624/184498621-4fd3a521-bd77-42a8-b277-eded579c45f2.png)
[참고 블로그](https://yumyumlog.tistory.com/244)

styled-components에서 css 헬퍼는 props의 조건에 따라 스타일을 다양하게 주는 경우에 보기 쉽게 하기 위해 사용한다고 합니다. 이 파일에서 css 헬퍼를 적용한 부분을 보면 조건에 따라 다양한 스타일을 주는 경우가 아니었기에 css 헬퍼를 사용하지 않고 styled만 사용하는 것은 어떨지 제안드립니다!
```suggestion
import styled from 'styled-components';
```